packages/core/src/tracing/hubextensions.ts

```diff
@@ -16,7 +17,7 @@ function traceHeaders(this: Hub): { [key: string]: string } {
 
   return span
     ? {
-        'sentry-trace': span.toTraceparent(),
+        'sentry-trace': spanToTraceHeader(span),
```

Comment:
(no action required)
TIL about this function. Not sure why it exists - my guess is downstream SDKs (maybe RN/Electron)...